atc/syslog/drainer.go

```diff
@@ -173,6 +173,16 @@ func (d *drainer) sendEvent(logger lager.Logger, build db.Build, syslog *Syslog,
 		ts = time.Unix(selectedWorkerEvent.Time, 0)
 		tag = build.SyslogTag(selectedWorkerEvent.Origin.ID)
 		message = fmt.Sprintf("selected worker: %s", selectedWorkerEvent.WorkerName)
+	case event.EventTypeStreamingVolume:
+		var streamingVolumeEvent event.StreamingVolume
+		err := json.Unmarshal(*ev.Data, &streamingVolumeEvent)
+		if err != nil {
+			logger.Error("failed-to-unmarshal", err)
+			return err
+		}
+		ts = time.Unix(streamingVolumeEvent.Time, 0)
+		tag = build.SyslogTag(streamingVolumeEvent.Origin.ID)
+		message = fmt.Sprintf("streaming volume %s from %s to %s", streamingVolumeEvent.Volume, streamingVolumeEvent.SourceWorker, streamingVolumeEvent.DestWorker)
```

Comment:
```suggestion
		message = fmt.Sprintf("streaming volume %s from worker %s to %s", streamingVolumeEvent.Volume, streamingVolumeEvent.SourceWorker, streamingVolumeEvent.DestWorker)
```
I like the way `streaming volume xxx` so i know `xxx` is a volume. Could we also do that for worker?

Another question is the `to %s` part, is there a case that destination worker is different to the worker in `selected worker` event just above? If not, I feel `streaming volume xxx from worker xxx` might be good enough.